src/util/stft.rs

```diff
@@ -463,7 +499,7 @@ impl<const NUM_SIDECHAIN_INPUTS: usize> StftHelper<NUM_SIDECHAIN_INPUTS> {
 
                         let remaining_padding = padding_to_copy - self.padding;
                         if remaining_padding > 0 {
-                            padding_buffer.copy_within(..padding_to_copy, 0);
+                            padding_buffer.copy_within(..self.padding, 0);
                         }
```

Comment:
This is incorrect. The old version was also incorrect, but the new version won't ever do anything. This should be:

```rust
let remaining_padding = self.padding - padding_to_copy;
if remaining_padding > 0 {
    padding_buffer.copy_within(padding_to_copy.., 0);
}
```

Or perhaps a bit easier to follow:

```rust
// Any remaining padding tail should be moved towards the start of the
// buffer
padding_buffer.copy_within(padding_to_copy.., 0);

// And we obviously don't want this to feedback
padding_buffer[self.padding - padding_to_copy..].fill(0.0);
```